draft-ietf-v6ops-464xlaton.xml

```diff
@@ -244,10 +245,13 @@ Despite the word "wireless" in the name, this architecure is applicable for wire
 </li>
 </ul>
 <t>
-In the latter case, the CLAT instance needs a single IPv6 and a single IPv4 address only.
+In the latter case, the CLAT instance needs a single IPv6 CLAT address (see <xref target="v6addr"/>) and a single IPv4 address only.
```

Comment:
```suggestion
In the single-address model, the CLAT instance needs a single IPv6 CLAT address (see <xref target="v6addr"/>) and a single IPv4 address only (distinct from the one or more IPv6 addresses used by the node running CLAT for its own native IPv6 connectivity).
```

The replacement of "latter case" is just for clarity.

The rest isn't strictly necessary, but I thought might be clearer to readers that "single address" clearly does not mean "IPv6 translation can reuse the /128 assigned to a device when the network admin does /128s" but rather "CLAT itself only needs a /128"